package-lock.json

```diff
@@ -1,8 +1,16451 @@
 {
   "name": "forkcms",
   "version": "1.0.0",
-  "lockfileVersion": 1,
+  "lockfileVersion": 2,
   "requires": true,
+  "packages": {
```

Comment:
why is there suddenly so much npm stuff here? seems to contain a bunch of things that arent related